fetch/src/post.ts

```diff
@@ -1,7 +1,6 @@
 // @ts-nocheck
-
 const { Schema, model } = require('mongoose');
-
+import CustomTag from 'server/src/models/customtag';
```

Comment:
This doesn't import properly on my machine

Reply:
fixed